Cargo.toml

```diff
@@ -42,7 +42,7 @@ futures = "0.3.30"
 pgrx-tests = "0.11.3"
 rstest = "0.19.0"
 serde_arrow = { version = "0.11.3", features = ["arrow-51"] }
-shared = { git = "https://github.com/paradedb/paradedb.git", rev = "4854652", features = ["fixtures"] }
+shared = { git = "https://github.com/paradedb/paradedb.git", branch = "add-util-record-batches", features = ["fixtures"] }
```

Comment:
I'm not sure it's good practice to point to a branch in released code. I think you should still pin to the commit.

Reply:
Yes, it will be reverted when the `shared` crate PR is merged. This is just temporary to make sure the CI is happy all the way.